ros_ws/src/control/CMakeLists.txt

```diff
@@ -16,8 +17,17 @@ catkin_package(
     LIBRARIES control
     CATKIN_DEPENDS geometry_msgs roscpp std_msgs sensor_msgs
     DEPENDS system_lib
+    CATKIN_DEPENDS message_runtime
 )
 
+################################################
+## Declare ROS messages, services and actions ##
+################################################
+## Generate message in the 'msg' folder
+add_message_files(FILES Encoder.msg)
+
+
```

Comment:
There is a missing function here which is what is causing the msg build failure.  Since this depends on std_msgs you need the line:

`generate_messages(DEPENDENCIES std_msgs)`

to come here.  This entire message block will also have to be moved up to be above the catkin_specific configuration block in order for it to build.